WordPress/build.gradle

```diff
@@ -8,7 +8,7 @@ buildscript {
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion"
         classpath 'com.google.gms:google-services:4.3.8'
         classpath 'se.bjurr.violations:violation-comments-to-github-gradle-plugin:1.67'
-        classpath 'io.sentry:sentry-android-gradle-plugin:1.7.28'
+        classpath 'io.sentry:sentry-android-gradle-plugin:2.0.0'
```

Comment:
I'm not sure if we want to update this plugin or remove it as in [this PR.](https://github.com/wordpress-mobile/WordPress-Android/pull/14798/files#diff-276cc53d974bad3212f8e1f709d32ea3f8d380c4dccb10419505d283fb41f2d7L553)

WDYT @oguzkocer ? I didn't test it but even if [we don't obfuscate but only minify ](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/build.gradle#L186)code on production, we would probably benefit from sending mapping files to Sentry.

Reply:
@wzieba That WPAndroid PR is not supposed to remove the plugin either. I've an unpublished commit where I was testing the obfuscation. I was worried about plugin not doing anything if the options are not set and I was comparing the old version behavior with and without the new plugin version. I didn't spend much time on it and where I landed was to just read the source code for the plugin to at least check what it's doing if no option is set.

While typing this comment, [I quickly had a look](https://github.com/getsentry/sentry-android-gradle-plugin/blob/main/plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt) and it looks like it's utilizing information from the android plugin, so we probably want to keep the plugin.

Anyhow, if you want to take over this work to update Sentry to the latest version, (and close that PR) you are more than welcome to do so. Actually, you'd be doing me a big favor 😄 🙇‍♂️

Reply:
Ok, gotcha thanks for the context.

> (...) it looks like it's utilizing information from the android plugin

True! 

> if you want to take over this work to update Sentry to the latest version, (and close that PR)

Thanks, yeah I'll willingly take care of this 😃